### PR TITLE
aovid error with widom insertion. exclusion radius should not have any effect here

### PR DESCRIPTION
**Pull request by jonaslandsgesell**

### CODE REVIEW

File: src/core/reaction_ensemble.cpp [hunk 1]
```diff
@@ -1792,11 +1791,7 @@ WidomInsertion::measure_excess_chemical_potential(int reaction_id) {
          // need to hide the particle and recover it
   make_reaction_attempt(current_reaction, changed_particles_properties,
                         p_ids_created_particles, hidden_particles_properties);
-  double E_pot_new;
-  if (particle_inserted_too_close_to_another_one == true)
-    E_pot_new = std::numeric_limits<double>::max();
-  else
-    E_pot_new = calculate_current_potential_energy_of_system();
+  double E_pot_new= calculate_current_potential_energy_of_system();
```
Comment (fweik) [inline src/core/reaction_ensemble.cpp]: Can be `const` now

**Comment by jonaslandsgesell** [comment]
LB test fails because it does not science the desired accuracy. Untouched.

**Comment by KaiSzuttor** [comment]
@jonaslandsgesell

**Comment by jonaslandsgesell** [comment]
This fix workes.

Am Sa., 3. Nov. 2018, 06:20 hat Kai Szuttor <notifications@github.com>
geschrieben:

**Comment by KaiSzuttor** [comment]
style check failed

**Comment by fweik** [comment]
@jonaslandsgesell what is the status on this?

**Pull request merged**